src/screens/FarmSearch/FarmSearch.js

```diff
@@ -29,15 +47,27 @@ class FarmSearch extends React.PureComponent {
 
         <div className="farm-search__grid">
           <NewFarmCard />
-          {details.map(id => {
+          {farms.map(farm => {
+            const {
+              farmId,
+              foodHubAffiliation,
+              farmName,
+              contactFirstName,
+              contactLastName,
+              address: farmAddress
+            } = farm;
+            const hubParticipant = Boolean(foodHubAffiliation);
+            const farmerName = `${contactFirstName} ${contactLastName}`;
             return (
               <FarmCard
-                key={id}
-                hubApproved={Boolean(1)}
+                key={farmId}
+                // TODO: refactor hubApproved => hubParticipant
```

Comment:
can we pass in the `farm` object as a prop and handle the formatting in the farm card itself? would make this grid a lot more concise :)